types/stylelint/index.d.ts

```diff
@@ -41,6 +41,7 @@ type EmptyResult = {
 			lang?: undefined;
```

Comment:
@jeddy3 can we remove `nodes` and `lang`?
cf https://github.com/stylelint/stylelint/commit/36e2dbb3fc1b42d670ddc97b55ee50f883c3f0d1#diff-9a4b261e295bef1ccce813aa194ac4023a1dcf98236667889964b1ba6fbc551fR56-R60

Reply:
I think they are no longer necessary. Even if they're removed, the type error doesn't happen. But I think we should remove in another PR.

Reply:
Just opened PR #8088 for the refactoring.